RELEASES.md

```diff
@@ -1,3 +1,97 @@
+Version 1.70.0 (2023-06-01)
+==========================
+
+<a id="1.70.0-Language"></a>
+
+Language
+--------
+- [Relax ordering rules for `asm!` operands](https://github.com/rust-lang/rust/pull/105798/)
+- [Properly allow macro expanded `format_args` invocations to uses captures](https://github.com/rust-lang/rust/pull/106505/)
+- [Lint ambiguous glob re-exports](https://github.com/rust-lang/rust/pull/107880/)
+- [Initial support for return type notation (RTN)](https://github.com/rust-lang/rust/pull/109010/)
```

Comment:
feature(return_type_notation) is unstable, so probably should not be mentioned in a stable release.